pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -200,6 +216,19 @@ protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndO
     }
   }
 
+  private void summarizeQuery(long requestId, BrokerResponse brokerResponse) {
+    ObjectNode stats = brokerResponse instanceof BrokerResponseNativeV2
+        ? ((BrokerResponseNativeV2) brokerResponse).getStageStats()
+        : JsonNodeFactory.instance.objectNode();
+    if (brokerResponse.getExceptions().isEmpty()) {
+      LOGGER.debug(MSE_STATS_MARKER, "Request {} finished in {}ms. Errors: {}. Stats: {}",
```

Comment:
Exception is always empty here

Reply:
I know, I just wanted to keep the logs stable when we have exceptions and when we don't to make it easy to consume these logs automatically, but I can also see advantages in having different log messages for each case. What do you think?

Reply:
I've modified this code slightly. Please take a new look. Now we indicate whether there was an error or not and the logging level can be configured with a query option